### PR TITLE
Sync with production

### PR DESCRIPTION
**Pull request by Krzemien97**

### CODE REVIEW

File: docusaurus.config.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ const config: Config = {
   organizationName: 'fictionlab', // Usually your GitHub org/user name.
   projectName: 'fictionlab.github.io', // Usually your repo name.
   deploymentBranch: 'gh-pages',
+  trailingSlash: false,
```
Comment (bjsowa) [inline docusaurus.config.ts:23]: What's the reasoning behind this change?

File: docusaurus.config.ts [hunk 2]
```diff
@@ -69,6 +70,12 @@ const config: Config = {
   ],
 
   themeConfig: {
+    algolia: {
+      appId: 'H6BESWXMON',
+      apiKey: '94b54b551433d306dade3f9cdb84b92b',
```
Comment (bjsowa) [inline docusaurus.config.ts:75]: Hope these are not secrets.

File: docusaurus.config.ts (hunk 1 above)
Comment (Krzemien97) [reply docusaurus.config.ts:23]: Right now all our pages have trailing slashes at the end, but sitemap has URLs without them. When Google crawls the site, it is redirected to site with trailing slash and the bot is not happy about it (refuse to index it).

File: docusaurus.config.ts (hunk 2 above)
Comment (Krzemien97) [reply docusaurus.config.ts:75]: These are public API keys, safe to use in frontend.

**Review by bjsowa — APPROVED** [review]

**Comment by bjsowa** [comment]
@Krzemien97 Is the search bar supposed to work?

**Comment by Krzemien97** [comment]
Algolia search needs like 1 day to index the site. I don't know if it should work in local environment.

**Pull request merged**